internal/grpc/resources.go

```diff
@@ -127,3 +129,25 @@ func (r routeConfigurationsByName) Swap(i, j int) { r[i], r[j] = r[j], r[i] }
 func (r routeConfigurationsByName) Less(i, j int) bool {
 	return r[i].(*v2.RouteConfiguration).Name < r[j].(*v2.RouteConfiguration).Name
 }
+
+// SDS implements the RDS v2 gRPC API.
+type SDS struct {
+	Cache
+}
+
+// Values returns a sorted list of RouteConfigurations.
+func (s *SDS) Values(filter func(string) bool) []proto.Message {
+	v := s.Cache.Values(filter)
+	sort.Stable(routeConfigurationsByName(v))
+	return v
+}
+
+func (s *SDS) TypeURL() string { return secretType }
+
+type secretsByName []proto.Message
```

Comment:
This breaks the staticcheck since it's not used:

```
internal/grpc/resources.go:147:6: type secretsByName is unused (U1000)
make: *** [Makefile:60: staticcheck] Error 1
```

Reply:
sneaky :).  sorry, let me fix that

Reply:
We just reenabled that staticcheck. Once you rebase CI will pick it up. =)

Reply:
There we go. :)

Reply:
Rebased!